services/localfile/client/infrastructure/input/file.utils.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2019 Snowflake Inc. All rights reserved.
+
+	Licensed under the Apache License, Version 2.0 (the
+	"License"); you may not use this file except in compliance
+	with the License.  You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing,
+	software distributed under the License is distributed on an
+	"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+	KIND, either express or implied.  See the License for the
+	specific language governing permissions and limitations
+	under the License.
+*/
+
+package input
+
+import (
+	"errors"
+	pb "github.com/Snowflake-Labs/sansshell/services/localfile"
+	"path/filepath"
+)
+
+func getFileTypeFromPath(filePath string) (pb.FileFormat, error) {
+	fileExt := filepath.Ext(filePath)
+
+	switch fileExt {
+	case ".yml":
+		return pb.FileFormat_YML, nil
+	case ".yaml":
```

Comment:
you can do `case ".yml", ".yaml"` instead

Reply:
done